src/Neo.CLI/Settings.cs

```diff
@@ -60,32 +61,44 @@ public Settings(IConfigurationSection section)
             P2P = new(section.GetSection(nameof(P2P)));
             UnlockWallet = new(section.GetSection(nameof(UnlockWallet)));
         }
+
+        public Settings()
+        {
+            Logger = new LoggerSettings();
```

Comment:
Contracts must be initialized

Reply:
This is not my code, it was here like this.